tests/model_fields/test_decimalfield.py

```diff
@@ -91,7 +91,10 @@ def test_lookup_really_big_value(self):
         Really big values can be used in a filter statement.
         """
         # This should not crash.
-        Foo.objects.filter(d__gte=100000000000)
+        self.assertQuerySetEqual(Foo.objects.filter(d__gte=100000000000), [])
```

Comment:
```suggestion
        self.assertSequenceEqual(Foo.objects.filter(d__gte=100000000000), [])
```

Reply:
Looks like assertQuerySetEqual() doesn't offer any extra value in this case 👍

Reply:
Actually assertSequenceEqual() does more than assertQuerySetEqual().

Just looking through the tests there are lots of examples of comparing querysets to [] with assertQuerySetEqual().

Can I ask why the recommended change? 🙂

Reply:
Because we compare to an empty list, there is no need to transform, to order etc. and `assertSequenceEqual()` implicitly call `list()`.

Reply:
See d795259ea96004df0a2469246229a146307bcd2c.

Reply:
In some cases we left `assertQuerySetEqual()` to avoid mixing `assertQuerySetEqual()` and `assertSequenceEqual()` in a single test method.

Reply:
`assertSequenceEqual()` and  `assertCountEqual()` are preferable. `assertQuerySetEqual()` is useful when you need to transform results.

Reply:
fair enough 🤷‍♂️